### PR TITLE
#164198494 create article 

### PR DESCRIPTION
**Pull request by danprocoder**
#### What does this PR do?
Adds feature to create a new article.

#### Description of Task to be completed?
1. Create tests for creating new article.
2. Save new article to database.

#### How should this be manually tested?
1. Clone this branch.
2. Run `npm install` to install the dependencies.
3. Run `npm test` to run the tests.

#### What are the relevant pivotal tracker stories?
[#164198494](https://pivotaltracker.com/story/show/164198494)

### CODE REVIEW

File: server/database/models/index.js [hunk 1]
```diff
@@ -39,4 +39,4 @@ Object.keys(db).forEach(modelName => {
 db.sequelize = sequelize;
 db.Sequelize = Sequelize;
 
-export default db;
+module.exports = db;
```
Comment (andela-gike) [inline server/database/models/index.js]: Why was this changed back to `module.exports = db`

Comment (danprocoder) [reply server/database/models/index.js]: @andela-gike
The model class was `undefined` when I imported it when I used `export default` but it worked when I changed it back to `module.exports`

**Review by andela-gike — APPROVED** [review]
